codalab/bundles/named_bundle.py

```diff
@@ -24,6 +24,9 @@ class NamedBundle(Bundle):
         MetadataSpec('created', int, 'Time when this bundle was created.', generated=True, formatting='date'),
         MetadataSpec('data_size', int, 'Size of this bundle (in bytes).', generated=True, formatting='size'),
         MetadataSpec('failure_message', str, 'Error message if this run bundle failed.', generated=True,),
+        MetadataSpec('link_url', str, 'Link URL of bundle.', optional=True),
```

Comment:
I needed to add these here so that we can have linked bundles that are non-UploadedBundles (we will need to do this anyway once run bundles upload their results to Azure Blob Storage).